.github/workflows/pythonpylint.yml

```diff
@@ -24,9 +24,13 @@ jobs:
                   pip install -r requirements.txt
                   pip install IPython
                   pip install pylint
+            - name: Install music21 in editable mode
+              run: |
+                  python -m pip install -e .
             - name: Lint with pylint
               run: |
                   pylint -j0 music21
+                  pylint -j0 documentation
```

Comment:
I'll revert if this feels like a waste of carbon. Installing music21 in editable mode was just how I got the documentation package to find the music21 package given how its imports are written.